introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Alfonso. I'm a sysadmin who likes to code and want to master git/github usage.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.